linux_os/guide/services/ntp/chronyd_specify_remote_server/bash/shared.sh

```diff
@@ -4,6 +4,13 @@
 
 config_file="{{{ chrony_conf_path }}}"
 
-if ! grep -q '^[\s]*(?:server|pool)[\s]+[\w]+' "$config_file" ; then
```

Comment:
Hello @rumch-se , I have analyzed the expression and I think that the only thing which needs to be fixed is this line (regarding Bash).
There should be:
```
if ! grep -q '^[[:space:]]*\(server\|pool\)[[:space:]]\+[[:graph:]]\+' "$config_file" ; then
```
The grep expression will then behave as we would expect.